app/views/api/messages/_message.xml.builder

```diff
@@ -0,0 +1,11 @@
+xml.tag! "message", :id => message.id,
+                    :from_user_id => message.from_user_id,
+                    :to_user_id => message.to_user_id,
+                    :sent_on => message.sent_on,
```

Comment:
Another thing where notes api differ from everything else is date formats. In other places dates look like this: [`2014-04-02T17:09:44Z`](https://www.openstreetmap.org/api/0.6/way/1000). In the notes api they look like this: [`2013-05-01 08:10:42 UTC`](https://www.openstreetmap.org/api/0.6/notes/1000).

Dates here are written like in the notes api.

Reply:
Thanks, I wasn't aware of this. The Z format is more appropriate for API, since it is easier to machine parse. Will modify.